libs/styles/_variables.scss

```diff
@@ -0,0 +1,70 @@
+//Color
+$primary: #00B207;
+$secondary: #84D187;
+$tertiary: #2C742F;
+$warning: #FF8A00;
+$danger: #EA4B48;
+$white: #FFFFFF;
+
+//Gray
+$gray-900: #1A1A1A;
+$gray-800: #333333;
+$gray-700: #4D4D4D;
+$gray-600: #666666;
+$gray-500: #828282;
+$gray-400: #999999;
+$gray-300: #B3B3B3;
+$gray-200: #CCCCCC;
+$gray-100: #E5E5E5;
+$gray-0: #F2F2F2;
+
+//Green gray
+$green-gray-900: #002603;
+$green-gray-800: #173B1A;
+$green-gray-700: #2B572E;
+$green-gray-600: #406B42;
+$green-gray-500: #618062;
+$green-gray-400: #7A997C;
+$green-gray-300: #96B297;
+$green-gray-200: #B4CCB4;
+$green-gray-100: #DAE5DA;
+$green-gray-0: #EDF2EE;
+
+//Heading
+$heading-1: 72px;
+$heading-2: 56px;
+$heading-3: 48px;
+$heading-4: 40px;
+$heading-5: 36px;
+$heading-6: 32px;
+
+//Body
+$font-size-tn: 12px;
+$font-size-sm: 14px;
+$font-size-md: 16px;
+$font-size-lg: 18px;
+$font-size-xl: 20px;
+$font-size-xxl: 24px;
+
+//Font weight
+$weight-300: 300;
+$weight-400: 400;
+$weight-500: 500;
+$weight-600: 600;
+$weight-700: 700;
+
+
+//Border radius
+$border-radius-button: 43px;
+$border-radius-button-rounded: 50%;
+$border-radius-card-primary: 6px;
+$border-radius-card-secondary: 8px;
+
+//Breakpoints
+$breakpoint-xs: 320px;
+$breakpoint-sm: 576px;
+$breakpoint-md: 768px;
+$breakpoint-lg: 1024px;
+$breakpoint-xl: 1280px;
+$breakpoint-xxl: 1440px;
+$breakpoint-xxxl: 1920px;
```

Comment:
Move to separate _breakpoints file, because we will not use breakpoints very often